File: Shelter.Core/Models/Animal.cs
namespace Shelter.Core.Models;

public class Animal
{
    
    public Guid AnimalId { get; set; }

    public string Image { get; set; } = string.Empty;
    
    public string Name { get; set; }
    
    public int Age { get; set; }
    
    public string Breed { get; set; }
    
    public string Description { get; set; }
    
    public List<Booking> Bookings { get; set; }
}

File: Shelter.Ng/src/models/animal.ts
import { AnimalStatus } from "../enum/animalStatus";

export class Animal {
    animalId : string = '';
    name : string = '';
    age : number = 0;
    breed : string = '';
    description : string = '';
    image: string | null = '';
    status: AnimalStatus = AnimalStatus.Available;
    

}

